mgplvm/kernels/stationary.py

```diff
@@ -121,11 +125,11 @@ def prms(self) -> Tuple[Tensor, Tensor]:
 
     @property
     def scale_sqr(self) -> Tensor:
-        return self._scale.square()
+        return self.scale.square()
```

Comment:
if `softplus` works better as a parameterization, maybe we should have `scale_sqr = softplus(self._scale)` and `scale = (self.scale_sqr+1E-20).sqrt()`? This way we won't needlessly call both softplus and square to ensure `scale_sqr` is positive?

Reply:
I'm happy with either

Reply:
alright I've done this now